### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by Redmonkeycloud**
New Folder containing all the tutorials from wiki.

### CODE REVIEW

**Comment by derevirn** [comment]
Rename the `Home.md` file to `README.md` so it is displayed automatically in the tutorials folder. Also, replace any references to Wiki with Tutorial.

File: Tutorials/Home.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Welcome to the OPEN-PROM wiki!
```
Comment (giannou) [inline Tutorials/Home.md:1]: this is not a Wiki anymore, suggested change: refer to the Turorials folder rather than the Wiki

**Review by giannou — CHANGES_REQUESTED** [review]
better have a sense of flow: numbered file names would help (see also here: https://github.com/remindmodel/remind/tree/develop/tutorials)

**Comment by Redmonkeycloud** [comment]
Derevirn's suggestions have been integrated!

**Comment by derevirn** [comment]
Links between tutorials need to have this format:

[link](https://github.com/e3modelling/OPEN-PROM/blob/main/Tutorials/File.md)

Also, spaces in links must be replaced with %20, or ideally remove them from file names entirely, and put underscores ( _ ) instead.

**Comment by Redmonkeycloud** [comment]
Anastasis suggestions about numbered file names added!

**Comment by Redmonkeycloud** [comment]
So all the links have been replaced with file names, the number filling system has been integrated and reworked and last but not least brief descriptions of all the files has been added in the README file.

**Review by giannou — APPROVED** [review]

**Pull request merged**